### PR TITLE
add a common-lint-fix make target to make golangci-lint fix found iss…

### PR DESCRIPTION
**Pull request by machine424**
…ues if it's supported by the linter

### CODE REVIEW

File: Makefile.common [hunk 1]
```diff
@@ -175,6 +175,13 @@ ifdef GOLANGCI_LINT
 	$(GOLANGCI_LINT) run $(GOLANGCI_LINT_OPTS) $(pkgs)
 endif
 
+.PHONY: common-lint-fix
```
Comment (machine424) [inline Makefile.common:178]: I'd like to get rid of the `$(GO) list...` and see what will happen.
I think it's a leftover from https://github.com/prometheus/prometheus/commit/f5d54e5be62072c0097fce439afa89ab3e399e4e
golangci-lint should take care of that now?

Comment (bboreham) [reply Makefile.common:178]: Agreed it should work without the `go list`.

Comment (machine424) [reply Makefile.common:175]: I removed that.

**Comment by beorn7** [comment]
Now we had a test failure that pretty much looked like "fail randomly for some reason not yet explained". I triggered a re-run, let's see what shows up this time.

**Comment by beorn7** [comment]
But now I saw that same random test failure somewhere else, so probably not related to the list remolav.

**Review by beorn7 — APPROVED** [review]

**Comment by machine424** [comment]
Thanks.
If you have some time I have another CI change here https://github.com/prometheus/prometheus/pull/13405

**Comment by beorn7** [comment]
#13405 is already handled by two other reviewers, I'll leave it to them.

**Pull request merged**